### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Fridrif27**
[DEMO LINK] (https://fridrif27.github.io/react_static-list-of-posts-js/)

### CODE REVIEW

**Review by StricTGod — APPROVED** [review]

File: src/App.jsx [hunk 1]
```diff
@@ -1,104 +1,19 @@
...
+import usersFromServer from './api/users.json';
+
+export const App = () => {
+  return (
+    <section className="App">
+      <h1 className="App__title">Static list of posts</h1>
+      <PostList
+        posts={postsFromServer}
+        users={usersFromServer}
+        comments={commentsFromServer}
```
Comment (BudnikOleksii) [inline src/App.jsx:15]: Prepare posts with user and comments and use it instead

File: src/components/CommentList/CommentList.jsx [hunk 2]
```diff
@@ -1 +1,16 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => {
+  if (!comments || comments.length === 0) {
```
Comment (BudnikOleksii) [inline src/components/CommentList/CommentList.jsx:5]: You don't need to render `CommentList` component at all if you don't have comments

File: src/components/CommentList/CommentList.jsx [hunk 3]
```diff
@@ -1 +1,16 @@
...
+    return <b data-cy="NoCommentsMessage">No comments yet</b>;
+  }
+
+  return (
+    <div className="CommentList">
+      {comments.map(c => (
+        <CommentInfo key={c.id} comment={c} />
```
Comment (BudnikOleksii) [inline src/components/CommentList/CommentList.jsx:12]: What is `c`? Avoid one-letter names

File: src/components/PostInfo/PostInfo.spec.jsx [hunk 4]
```diff
@@ -62,22 +62,22 @@ describe('PostInfo', () => {
...
+    //   cy.get('.CommentInfo').should('have.length', 3);
+    //   cy.get('.CommentInfo__name')
+    //     .eq(0)
+    //     .should('have.text', 'esse autem dolorum');
+    //   cy.get('.CommentInfo__name')
+    //     .eq(2)
+    //     .should(
+    //       'have.text',
+    //       'culpa eius tempora sit consequatur neque iure deserunt',
+    //     );
```
Comment (BudnikOleksii) [inline src/components/PostInfo/PostInfo.spec.jsx:75]: Uncomment all tests

**Review by BudnikOleksii — CHANGES_REQUESTED** [review]
Uncomment tests and fix your solution in accordance

**Pull request closed without merging**